contracts/src/v0.8/functions/v1_1_0/libraries/L1Block.sol

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This is a vendored contract, not a library.

In ChainSpecific we are calling the deployed contract, so really we only need an interface.
But the full contract may be useful as a test mock.